alerting/src/utils/analysis-utils.ts

```diff
@@ -1,11 +1,14 @@
 import fetch from 'node-fetch';
 import { get, isNil } from 'lodash';
 import bbox from '@turf/bbox';
+import { createGetCoverageUrl } from 'prism-common';
 import { Extent } from './raster-utils';
-import { getWCSLayerUrl } from './server-utils';
 import { ANALYSIS_API_URL } from '../constants';
 import { Alert } from '../entities/alerts.entity';
 
+// eslint-disable-next-line fp/no-mutation
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```

Comment:
Why is this change needed here?

Reply:
Good question.  I'm not sure.  I just copied it over from the server-utils.ts file.  I see that it was originally added with this PR: https://github.com/WFP-VAM/prism-app/pull/149/files#diff-1f860e81328aef6dfd726374f959ffb6ff8fa811ba25c9ca7fba8c665770d365R9

The direct reason is I was trying to preserve any prior functionality.  I think because analysis-utils imported from server-utils, NODE_TLS_REJECT_UNAUTHORIZED was essentially set to '0' inside analysis-utils, too.

The only original reason I can think of is that one of the WCS services had a certificate issue, so making https requests was rejecting and this was added as a workaround.  It might no longer be necessary.

Let me know if you'd like me to remove it.